sopel/coretasks.py

```diff
@@ -985,6 +1097,7 @@ def blocks(bot, trigger):
 
 @module.event('ACCOUNT')
 def account_notify(bot, trigger):
+    """Track users's account."""
```

Comment:
```suggestion
    """Track users' accounts."""
```

Is this a :fr: thing? 😹